genesyscloud/responsemanagement_response/resource_genesyscloud_responsemanagement_response_test.go

```diff
@@ -506,3 +512,42 @@ func testVerifyResponseManagementResponseDestroyed(state *terraform.State) error
 	// Success. All responses destroyed
 	return nil
 }
+
+func cleanupResponseAssets(folderName string) error {
+	var (
+		name    = "name"
+		fields  = []string{name}
+		varType = "STARTS_WITH"
+	)
+	config, err := provider.AuthorizeSdk()
+	if err != nil {
+		return err
+	}
```

Comment:
If we're not going to handle the error anywhere we call this function, we might as well just log this error here and `return` instead of `return err`

Reply:
Logged the error